extras/videoDrivers/SDL/VideoSDL.cpp

```diff
@@ -20,10 +20,12 @@
 #include "driver/VideoDriverLoaderInterface.h"
 #include "driver/VideoInterface.h"
 #include "helpers/containerUtils.h"
-#include <memory>
+#include "makeException.h"
 #include <boost/nowide/iostream.hpp>
+#include <boost/system/windows_error.hpp>
```

Comment:
why is the windows_error.hpp included here? I can't see anything related with it?

Reply:
Removed